pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel", "numpy==1.19.2", "Cython==0.29.17"]
+build-backend = "setuptools.build_meta"
```

Comment:
I take it this update does most, if not all, of the heavy lifting?

Reply:
@CosmoMatt yes and the MANIFEST.in allows the setup.py ro read in the pip readme and core requirements to prevent hardcoding each, making it eaiser to maintain in future versions